### DISCUSSION TITLE
radical ideas about geography

### DISCUSSION THREAD

**Discussion by dustymc** [ArctosDB/arctos#7666] [Ideas]
See https://github.com/ArctosDB/arctos/issues/7660, https://github.com/ArctosDB/arctos/issues/7374, etc., etc.

Arctos geography is standardized, but I think maybe that's not actually what anyone wants, and constraining assertions to spatial parents was (to my great surprise!) outright rejected. Maybe we should try something else.

One option might be to make table geog_auth_rec a free-for-all (or something simpler, or ????????), and I'll find a new place to do cool stuff with spatial/standardized data. 

I'm up for absolutely anything, tell me what ya'll want to do and I'll suggest a model which can do that.

@amgunderson @cjconroy @mkoo @genevieve-anderegg

**Comment by cjconroy** [comment]
I can't follow the entire thread. There is some higher level database thing you guys want to do that is beyond me. I just know that if I search for all mammals from Alaska, I want all the ones on the Alaskan islands and the mainland. Same for Mexican islands. I truly don't understand how Isla Santa Catalina in the Sea of Cortez now has a specific locality of "Isla Santa Catalina, Gulf of California Islands, Isla Santa Catalina, Gulf of California Islands, Baja California Sur". And no mention of Mexico. The island is in the Gulf, but the mice are from the island. Keeping track of political units is also useful due to permits.
I just realized that I came up with numbers for the RANGES project based on querying by country. I must have missed a bunch since I queried by terms like Alaska, Mexico, etc. Other than a spatial query (which only gets georeferenced specimens), how would I have pulled together a list of all our Mexican mammal holdings?

**Comment by dustymc** [comment]
>can't follow

That is my biggest concern about doing ANYTHING at this point; I thought we were all in unity on the last move, and here we are...

>database thing

No, the last move was to standardize, to follow  external "authorities" The "database thing" was only that we finally got the tools to do that, which we'd been talking about for a decade or two, properly.

> useful due to permits

I don't think any "authority" is going to confound geography and politics enough to meet that need (nor a few other things recently mentioned), hence my (radical!) suggestion that perhaps "curatorial geography" cannot have any rules at all. There are probably a dozen views of "California" for permitting purposes , for example. (And we could make spatial data of that, but I'm not convinced anyone could ***use*** it.)

>only gets georeferenced specimens

Georeferencing everything is part of what I thought we were unified on, but there was widespread rejection of the automation.

>Mexican mammal holdings

I'd need an example (and I'm afraid it would just be a distraction to this), but I suspect "Mexico" would be the proper geography assertion for anything on a Mexican island.

 I don't think this should be framed as a technical problem; I'd not have supported what we did if I didn't think I had a pretty solid mechanism for answering any sort of 'how can we..." question involving standardized data. I think this is entirely a social issue; tell me what you want to do and I'll address the technicalities.

**Comment by dustymc** [comment]
Briefly discussed with @mkoo , possibly do something with locality attributes (eg push something to flat.higher_geog, or ???????????).

**Reply by genevieve-anderegg** [reply]
Seeming more and more like we can create locality attributes that will assist with curatorial needs within our current GADM-framework (and with our current spatial restrictions) but discussion very much needed. I would love input from more collections and users in our committee!

Just realized I am a terrible person and did not update issues after our last meeting... off to do that now ....

**Comment by dustymc** [comment]
https://github.com/ArctosDB/arctos/issues/7374 is going forward, it will denormalize sea/overlap with IHO, and we should expect this to confound people; they will find one, not the other, and leave without what they came for. This should be somehow addressed.

https://docs.google.com/document/d/1tVmulmkFzN7P_IliFkeiOvMfKTAB_CCzG55ZxX0NFMI/edit#heading=h.3jdyx487bg21

Committee also maybe-kinda suggests GADM (and maybe IHO and probably everyone else...) has a too-limited/structured/something view of geography, perhaps a looser (more administrative?) model would be more appropriate.

**Comment by dustymc** [comment]
Some semi-arbitrary comments, originating in part near https://github.com/ArctosDB/arctos/issues/8269:

Clean functional requirements seem increasingly (from a baseline of "absolutely"!!) necessary if I'm to have any chance at all of suggesting a model which does whatever it is that The Community wants to do.

I think the current model is shaping many of the comments in various place-related issues, and these are resulting in what to me seems like technically nonviable situations. I very strongly suggest starting with a clean slate if anything more than minor evolutionary changes are desired.

Possibly the model is fine (would still be nice to have a functional requirements analysis to back that up if so) but the data need expansion. For example from "waters outside of the MA polygon ... are still part of MA!" - then surely they recognize that and publish data which supports this?? Is that particular problem simply a matter of Arctos finding such data, and does that scale if so?

In the complete opposite direction of the previous comment: "decided to not try and link different watery places with shape files." See my initial comments above, I still wonder if spatial operations should be entirely behind the scenes of some simplified geography model. The ultimate simplification (which may or may not be what anyone wants, I'm only hoping to inspire functional requirements here) would be removing geography altogether: locality attributes are fully capable of simultaneously saying "Atlantic Ocean" and "Massachusetts," and being crafted such that these assertions cannot be seen as erroneous regardless of any other data.

**Comment by happiah-madson** [comment]
>then surely they recognize that and publish data which supports this?? Is that particular problem simply a matter of Arctos finding such data, and does that scale if so?

https://www.mass.gov/info-details/massgis-data-ocean-mask

https://www.mass.gov/info-details/massgis-data-the-massachusetts-coastal-zone

**Comment by dustymc** [comment]
Discussed in https://docs.google.com/document/d/1aEzwbnDObmnjKkkyM9BVMqWhKACPloZenfc9kpMHxHo/edit?tab=t.0 - we are in the exploratory phases (@mkoo  and @genevieve-anderegg have data) of taking the first step towards what I've hinted at above with "watery geography." That (assuming it proceeds as hoped) may result in consideration of doing the same with dirt-n-rock geography, in which case Arctos would have a fully separated system (possibly there's the elusive functional requirement: separation!) in which

1. The collections assert whatever they want (as strings). This would hopefully be guided (probably not strictly controlled, as is GADM) by some external source, and there would be many details to work out (multiple assertions or combo-terms? Just term-control or structure? etc., etc.). I think the current consensus is that locality attributes are the mechanism for this (they offer a familiar and existing UI, no cardinality restrictions, and metadata), but this could also be implemented in some other way (eg like pre-spatial geography).
2. "Geography" (possibly under a new name, all details would be open to discussion) would remain as a wholly separated, entirely background process (useful for things like discovery, never available for assertion/to pick as part of entry). 

This would be a significant simplification for everyone. 
* Collections would have only to pick (zero or more, probably, and in a manner to be determined) 'place-things' (name also TBD). No clutter, no mix of assertable and non-assertable things, no complete reliance on an external source, just choose whatever locally-managed data you wish (and help manage it if you want) and rock on. (And no possibility of the aforementioned denormalization: There would be exactly one place from which to assert any place-term.) 
* I would no longer need to struggle with the separation between directly and indirectly asserted and derived and what's "data" and what's not-quite and etc.; there'd be a clear line between 'data' and 'maybe useful for some things but is not from the collection.' I'd not need to worry about updating any terms; spatial data could freely evolve without any implications of change in the collections, assertions would be of no concern to a spatial system.

For the example above, "Massachusetts" could mean (hopefully supported by the term definition, but likely NOT supported by a shapefile) whatever it's useful to mean, and could not be considered "wrong" based on any other data (such as coordinates). "Massachusetts" could be defined to include "The Massachusetts Coastal Zone," or those could be separate (and either, neither, or both asserted on any record), or "Massachusetts" and "Massachusetts including The Massachusetts Coastal Zone" could both exist, or WHATEVER - this would all be the domain of the collections.

The spatial system (which no user need know exists) could include eg MA according to GADM and 
<img width="275" alt="Screenshot 2024-11-08 at 07 57 40" src="https://github.com/user-attachments/assets/a2303609-e0dc-4667-9b10-6e42621d6c24"> from above, and a spatial search for "Massachusetts" would find records whose coordinates overlap either, entirely independently of any assertions. (And possibly more-precise spatial operations could be supported, but they'd still never impinge on what's "data" - only use it to add accessibility.)

I will try to keep this up to date with any discoveries made during the watery collections lead explorations, and we can revisit and decide how and if to proceed with land-based geography after we have some experience with watery. (Keeping in mind that the watery experiment will not be isolated from the complexity of the current system; a further progression would also be a further simplification, assuming nothing too unexpected turns up along the way.)

**Reply by genevieve-anderegg** [reply]
Excellent!!

**Reply by happiah-madson** [reply]
+1
